### ISSUE TITLE
Document difference of type attribute with jsonschema spec

### ISSUE THREAD

**Issue by casualjim** [OAI/OpenAPI-Specification#397]
The swagger specification in a definition doesn't allow for things like

``` json
{ "type": ["string", "null"] }
```

But the specification markdown document currently says that the definition is taken from json schema.
That is incorrect.

**Comment by Cwellan** [comment]
This thing is now allowed, and JSON schema Draft 4 allows it:
[JSON Schema Draft 4 Link](https://tools.ietf.org/html/draft-fge-json-schema-validation-00#page-15)
The issue must be closed.

**Comment by MikeRalphson** [comment]
Neither OpenAPI 2.0 or 3.0.x support unmodified JSON Schema Draft 04 schema objects and they specifically do **not** support the `type` property taking an `array`. In OpenAPI 3.0.x this must be represented as:

```yaml
type: 'string'
nullable: true
```

**Comment by Cwellan** [comment]
Why do you say that it specifically doesn't support the array ? How did you get the information ? 
The only thing I was able to find is:

> The following properties are taken directly from the JSON Schema definition and follow the same specifications:
> 
> -  ...
> - type

**Comment by MikeRalphson** [comment]
If you are referring to OpenAPI 2.0 then it is a long tortuous trail, but is summed up by [this comment](https://github.com/OAI/OpenAPI-Specification/issues/229#issuecomment-164592837). A published errata document would certainly help in this case.

Regarding OpenAPI 3.0.x

> The following properties are taken from the JSON Schema definition but their definitions were adjusted to the OpenAPI Specification.
>
>    type - Value MUST be a string. Multiple types via an array are not supported.

**Comment by philsturgeon** [comment]
This was fixed in #1977 so we can close this.
